client/src/components/commonComponents/ChatHeader/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {Appbar} from 'react-native-paper';
+import {Button} from 'react-native-paper';
+
+import style from './style.js';
+const ChatHeader = ({contectName, showGroupIcon}) => {
+  return (
+    <Appbar.Header style={style.container}>
+      <Icon.Button
+        name="chevron-left"
+        style={style.backicon}
+        onPress={() => {}}
```

Comment:
if this line doesn't do anything right now we can delete it, or put a comment to describe why it got an empty function